pyrdfj2/j2_functions.py

```diff
@@ -0,0 +1,202 @@
+import re
+from collections.abc import Iterable
+from datetime import date, datetime
+
+from dateutil import parser
+from uritemplate import URITemplate
+
+
+class Functions:
+    _cache = dict()
+
+    @staticmethod
+    def all():
+        return {
+            # TODO -- remove this in 0.2.0 -- deprecated use, now use
+            #   `| ttl(...)` instead
+            "ttl_fmt": turtle_format,
```

Comment:
since these were moved from functions to filters in the pysubyt lifetime

we should now get rid of them as functions
(we only kept it as a transition help)